RubeesEat/wwwroot/css/site.css

```diff
@@ -20,29 +23,6 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
     --secondary-color: #2d332f;
 }
 
-*, *::before, *::after {
-    box-sizing: border-box;
-}
-
-body {
-    line-height: 1.5;
-    -webkit-font-smoothing: antialiased;
-    margin: 0;
-    font-family: "Metric Regular", Arial, Tahoma, sans-serif;
-    text-align: center;
-    background-color: var(--secondary-color);
-    color: white;
-}
-
-img, picture, video, canvas, svg {
-    display: block;
-    max-width: 100%;
-}
-
-input, button, textarea, select {
-    font: inherit;
-}
-
 nav {
```

Comment:
Bootstrap has a nav component so we can convert this to bootstrap as well and remove our custom navbar